File: leetcode/contest/5249.py
from collections import stac

class Solution:
    def minRemoveToMakeValid(self, s: str) -> str:
        stack = []
        for letter in s:
            if letter != ")":
                stack.append(letter)
            if letter == ')':
                while True:
                    last = stack.pop



if __name__ == "__main__":
    input = "))(("

    print(Solution().minRemoveToMakeValid(input))

File: leetcode/contest/5249.cpp
#include <string>
#include <stack>

using namespace std;

class Solution {
    stack<char> prior;
public:
    string minRemoveToMakeValid(string s) {
        string temp;
        for (auto letter : s) {
            if (letter != ')')
            {
                prior.push(letter);
            } else
            {
                temp = get_substring();
            }
            
        }
    }

    string get_substring();
};